src/connectors/tests/elastic.rs

```diff
@@ -1097,6 +1099,7 @@ async fn elastic_https() -> Result<()> {
             .with_env_var("ELASTIC_PASSWORD", password)
             .with_env_var("xpack.security.enabled", "true")
             .with_env_var("xpack.security.http.ssl.enabled", "true")
+            .with_env_var("ingest.geoip.downloader.enabled", "false")
```

Comment:
Well this is surprising - would love to know what we triggered setting this to false!

Reply:
it defaults to true which means that as part of the startup ES will try to download GEOIP data for every test - that wastes a good bit of time / bandwidth / and can timeout when the container can't reach the internet. Since we don't use geoip data whacking that made sense to speed up tests :D